src/pages/catalog/elements/index.tsx

```diff
@@ -0,0 +1,153 @@
+          </p>
+        )}
+      </div>
+    </article>
+  );
+};
+
+export default function Courses() {
+  const [showRequestCourseModal, setShowRequestCourseModal] =
+    React.useState(false);
+  return (
+    <React.Fragment>
+      <Head>
+        <title>OER Question Elements | PrairieLearn</title>
+      </Head>
+
+      <PageBanner
+        title="Open Educational Resources (OER): Question Elements"
+        subtitle="A catalog of building blocks that allow you to create richer questions"
+      />
+
+      <div className="container-fluid my-3">
+        <div className="container-md">
+          <div className="alert alert-secondary mb-0">
+            <p>
+              The shared question elements on this page are provided as-is and
+              not maintained by PrairieLearn. They are only recommended for
+              instructors who are already familiar with the platform. To use an
+              element, you will have to access its Github repository and
+              manually copy the element into your course.
+            </p>
+          </div>
+        </div>
+      </div>
+
+      <div className="container-fluid my-5">
+        <div className="container-md">
+          <Heading>Publicly Shared Elements</Heading>
+          <div className="row">
+            <p>
+              This catalog contains question elements that have been created and
+              shared by students and instructors from various universities. You
+              can try them out by clicking the link to a demo assessment on
+              PrairieLearn. The assessment also contains instructions how you
+              can copy an element into your course if you like it. You can find
+              the link to each element&apos;s Github repository below.
+            </p>
+          </div>
+        </div>
+      </div>
+
+      <div className="container-fluid my-5">
+        <div className="container-md">
+          <div className={styles.grid}>
+            <SharedElementCard
+              image={oerTruthTable}
+              title="Truth Table"
+              href="https://us.prairielearn.com/pl/course_instance/169837/assessment/2489207"
+              github="https://github.com/PrairieLearn/pl-oer-element-truthtable"
+              ownerName="Pai Zheng and Yujie Miao"
+            />
+            <SharedElementCard
+              image={oerChemInput}
+              title="Chemistry Input"
+              href="https://us.prairielearn.com/pl/course_instance/169804/assessment/2489141"
+              github="https://github.com/PrairieLearn/pl-oer-element-cheminput"
+              ownerName="Ananya Barman, Jayshri Ganguli and Shengyuan Wang"
+            />
+          </div>
+        </div>
+      </div>
+
+      <div className="container-fluid my-5">
+        <div className="container-md">
+          <h2 className="h4">License</h2>
+          <p>All content here is made made available under an open license.</p>
```

Comment:
The GitHub repositories should have a [https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository] stating what exactly the license is.

Reply:
Added for the already listed OER repos (and will add to the future ones). Just wanted to bring up though that I copied this note from the other OER page and that (as far as I can see) none of the other OER repos have a license file.